json-schema-registry/src/main/java/io/micronaut/jsonschema/registry/types/Responses.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2017-2025 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.jsonschema.registry.types;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.serde.annotation.Serdeable;
+
+/**
+ * Response types for the JSON Schema Registry Client.
+ */
+@Serdeable
+public interface Responses {
```

Comment:
Move these out of the `Responses` class and you can put everything in `io.micronaut.jsonschema.registry.model` for consistency.